desktop/install/linux-install.md

```diff
@@ -135,7 +137,7 @@ Log out and log back in so that your group membership is re-evaluated.
     >
     > If you do not agree to the terms, the Docker Desktop application will close and  you can no longer run Docker Desktop on your machine. You can choose to accept the terms at a later date by opening Docker Desktop.
     {: .important}
-
+ps-dd4l-add-filesharing-docs
```

Comment:
that's weird, other than that LGTM

Reply:
emmm, tbh I have no idea how it got there

Reply:
done :)